src/app.js

```diff
@@ -96,7 +96,6 @@ class Messenger extends EventEmitter {
     this.app.engine('handlebars', exphbs({defaultLayout: 'main'}));
     this.app.set('view engine', 'handlebars');
 
-    this.app.use(bodyParser.json({ verify: this.verifyRequestSignature.bind(this) }));
     this.app.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
I'm not sure if this middleware should be specific to the route too. I say leave it as is.

docs: https://github.com/expressjs/body-parser#extended